test/common/bond_test.js

```diff
@@ -32,7 +36,7 @@ test('determineBondProvider should be "dcf-fence" if the URL host is "dg.foo"',
 });
 
 test('determineBondProvider should be "HCA" if the URL host ends with ".humancellatlas.org"', (t) => {
```

Comment:
please update the description of the test to reflect the new logic
e.g.
```suggestion
test('determineBondProvider should not return a provider if the URL host ends with ".humancellatlas.org"', (t) => {
```